hera_cal/tests/mock_uvdata.py

```diff
@@ -0,0 +1,284 @@
+"""Functions for mocking UVdata objects for testing purposes."""
+from __future__ import annotations
+
+from pyuvdata import UVData, UVCal
+from pyuvdata.uvdata import FastUVH5Meta
+from hera_cal import utils
+import numpy as np
+from astropy.coordinates import EarthLocation
+import yaml
+from hera_cal.data import DATA_PATH
+from pathlib import Path
+from hera_cal.red_groups import RedundantGroups
+
+HERA_LOC = EarthLocation.from_geodetic(
+    lat=-30.721526120690307,
+    lon=21.428303826863015,
+    height=1051.690000070259,
+)
```

Comment:
Normally we load this from this from pyuvdata's `KNOWN_TELESCOPES` so that we don't need to have duplicate sources of hera's position

Reply:
Oh, I didn't realize that existed! I've updated it.